manticore/core/cpu/abstractcpu.py

```diff
@@ -360,12 +360,22 @@ def _wrap_operands(self, operands):
 
     def decode_instruction(self, pc):
         '''
-        This will decode an instruction from memory pointed by @pc
+        This will decode an instruction from memory pointed by `pc` and store
+        it in self.instruction.
 
         :param int pc: address of the instruction
+        :return: None
```

Comment:
rm this? i think it's safe to assume if the docs don't specify a ret there is none